llvm/lib/Target/SBF/SBFFrameLowering.cpp

```diff
@@ -16,22 +16,23 @@
 #include "llvm/CodeGen/MachineFrameInfo.h"
 #include "llvm/CodeGen/MachineFunction.h"
 #include "llvm/CodeGen/MachineInstrBuilder.h"
-#include "llvm/CodeGen/MachineRegisterInfo.h"
 
 using namespace llvm;
 
 namespace {
 
 void adjustStackPointer(MachineFunction &MF, MachineBasicBlock &MBB,
                         MachineBasicBlock::iterator &MBBI,
-                        unsigned int Opcode) {
+                        bool Subtract) {
   MachineFrameInfo &MFI = MF.getFrameInfo();
   int NumBytes = (int)MFI.getStackSize();
+  if (Subtract)
+    NumBytes = - NumBytes;
```

Comment:
Nit: Perhaps instead of adding this if-statement, use the ternary operator below in the `addImm`.